src/porepy/viz/exporter.py

```diff
@@ -146,6 +147,12 @@ def __init__(
         )
         """Flag controlling whether constant data is exported to a separate file."""
 
+        self._length_scale: float = length_scale
+        """Length scale for the grid. All coordinates are multiplied by this value
+        before exporting.
+
+        """
```

Comment:
OK. Though, using a 'dim'-dimensional array wouldn't be such a big deal either, thinking of overly thin media for instance. But this can also be postponed until someone sees need for it.

Reply:
I think your suggestion is an interesting one. But I don't think it's relevant in the context of this PR. Doing this on a run script level with inheritance from Exporter should be very doable in something like 30 lines.